fast-list.js

```diff
@@ -72,42 +63,108 @@ function FastList(source) {
   on(this.els.container, 'click', this);
   on(window, 'resize', this);
 
-  this.rendered = schedule.mutation(function() {
-    this.updateContainerGeometry();
-    this.updateListHeight();
+  // Create a list element if one wasn't provided
+  if (!this.els.list) this.els.list = document.createElement('ul');
+  if (!this.els.itemContainer) this.els.itemContainer = this.els.list;
+
+  this.rendered = schedule
+    .mutation(this.renderPhase1.bind(this))
+    .then(this.renderPhase2.bind(this))
+    .then(function() {
+
```

Comment:
oh I thought you left me a comment on IRC yesterday night about an issue like this.
We're setting the height + scroll in phase1 but the scroll listener in phase 2.
So we have 360ms where the user could scroll no?
I'm fine with just setting the height/scroll in phase 2 too.